src/render/quantum.rs

```diff
@@ -75,9 +75,12 @@ impl AllocInner {
 
 /// Render thread channel buffer
```

Comment:
`AudioRenderQuantumChannel` now has no public methods, except the `Deref(Mut) to [f32]`